libraries/AP_RangeFinder/AP_RangeFinder_trone.cpp

```diff
@@ -0,0 +1,156 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/*
+  driver for trone rangefinder
+ */
+#include "AP_RangeFinder_trone.h"
+
+#include <utility>
+#include <AP_HAL/AP_HAL.h>
+
+extern const AP_HAL::HAL& hal;
+
+#define TRONE_I2C_ADDR 0x30
+
+// registers
+#define TRONE_MEASURE 0x00
+#define TRONE_WHOAMI  0x01
+#define TRONE_WHOAMI_VALUE 0xA1
```

Comment:
they should be used, I'll fix. I don't have a trone to test with btw, I'm relying on Luis to test